build.yaml

```diff
@@ -717,6 +717,18 @@ common_cli:
     wget https://raw.githubusercontent.com/AndrewDryga/vagrant-box-osx/master/Vagrantfile
     vagrant up
 
+  run_lint: &run_lint |
+    # we need to unset PIP_INDEX_URL since it causes tox to fail.
+    # in order to use our pip caching we need to append --trusted-hosts
+    # which we currently can't do through tox.
+    unset PIP_INDEX_URL
+    # run flake8 lint tests on Flocker source code
+    tox -e lint
+
+  install_flake8: &install_flake8 |
+    # flake8 is used by lint tests on Flocker source code
+    \${PARSE_LOGS} pip install flake8 \${PIP_ADDITIONAL_OPTIONS}
```

Comment:
do we need tox ?
I think this might be the only place where we call tox for running tests